### DISCUSSION TITLE
FR: Set `amp=true` for all pages via `next.js.config`

### DISCUSSION THREAD

**Discussion by derolf** [vercel/next.js#10649] [Help]
I have a website where all pages are amp pages. Currently, I need to mark every single page with:

    export const config = { amp: true };

I was unable to find a way to set this globally for all pages. So, it would be nice to specify `amp: true` in `next.js.config`.

**Comment by timneutkens** [comment]
You have to opt-in per page as all pages have to be compatible with AMP rendering when you enable it.

**Comment by derolf** [comment]
That’s what I meant: all my pages are AMP ready.

Also, I could just opt-out of pages that are not by exporting amp=false in their config.

**Comment by wernerpj1** [comment]
Man i've putted in all pages from my nextjs i dont know what im losing here running on an A statically generated blog example using Next.js, Markdown, and TypeScript template from vercel can anyone help?
